js/background.js

```diff
@@ -972,6 +973,9 @@
       return toastID;
     };
 
+    // Get memberlist. This function is not accurate >>
+    // window.getMemberList = window.lokiPublicChatAPI.getListOfMembers();
+
```

Comment:
to be removed, right?

Reply:
No. For future implementation from Maxim.